doc/operator-user-guide.md

```diff
@@ -313,6 +315,137 @@ Note that S3 secret name is provided directly in the APIManager custom resource.
 
 Check [*APIManager SystemS3Spec*](apimanager-reference.md#SystemS3Spec) for reference.
 
+_For AWS STS ApiManager CR contains **STS: true** field._
+Check [AWS STS Support](#aws-sts-support) for more details.
+
+##### AWS STS Support
+
+With the new support for STS authentication (Secure Token Service for  short-term, limited-privilege security credentials),
+the secret generated by the Cloud Credential tooling looks differ from IAM (Identity and Access Management) secret.
+There are two new fields AWS_ROLE_ARN and AWS_WEB_IDENTITY_TOKEN_FILE are present instead of
+AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY.  
+Below are samples for STS and IAM secrets:
+
+**STS Secret sample**
+```yaml
+kind: Secret
+apiVersion: v1
+metadata: 
+  name: s3-credentials
+  namespace: 3scale-test
+data: 
+  AWS_ROLE_ARN: XXXXX=
+  AWS_WEB_IDENTITY_TOKEN_FILE: XXXXX=
+  AWS_BUCKET: XXXXX=
+  AWS_REGION: XXX
+type: Opaque
+```
+
+**IAM Secret sample**
+```yaml
+kind: Secret
+apiVersion: v1
+metadata: 
+  name: s3-credentials
+  namespace: 3scale-test
+data: 
+  AWS_ACCESS_KEY_ID: XXXXX=
+  AWS_SECRET_ACCESS_KEY: XXXXX=
+  AWS_BUCKET: XXXXX=
+  AWS_REGION: XXX
+type: Opaque
+```
+
+**Summary for keys for each secret "type"**
+
+|Secret key                 |Required for IAM|Required for STS|
+|---------------------------|---|---|
+AWS_ACCESS_KEY_ID           |Y|N|
+AWS_SECRET_ACCESS_KEY       |Y|N|
+AWS_ROLE_ARN                |N|Y|
+AWS_WEB_IDENTITY_TOKEN_FILE |N|Y|
+AWS_BUCKET                  |Y|Y|
+AWS_REGION                  |Y|Y|
+AWS_HOSTNAME                |N|N|
+AWS_PROTOCOL                |N|N|
+AWS_PATH_STYLE              |N|N|
+
+
+
+**SystemApp Environment Variables**  
```

Comment:
No need to expose the system app env vars at the user level. This is internal stuff.